### PR TITLE
En/Decode rule status

### PR DESCRIPTION
**Pull request by jcc333**

### CODE REVIEW

File: tests/test_decoder.py [hunk 1]
```diff
@@ -142,6 +143,7 @@ def test_decoder_for_rule(self):
         self.assertEquals(decoded.alert_by, 'any')
         self.assertEquals(decoded.key, 'foo')
         self.assertEquals(len(decoded.conditions), 1)
+        self.assertEquals(decoded.status, 'logonly')
```
Comment (mrgaaron) [inline tests/test_decoder.py]: If we're going to the trouble to provide constants, shouldn't the value decode to one of those constants as opposed to a string?

**Comment by mrgaaron** [comment]
To make this work where you really need it to, you need to change the code in `decode_rule_list` in the `TempoIQDecoder` class.  This will require a backend change eventually as well, to actually include the status in the legato output for that endpoint.  The whole thing is janky anyway.  It shouldn't return a Map<String, String> but a sequence of Rules with whatever information we want to put in them.

File: tests/test_decoder.py (hunk 1 above)
Comment (jcc333) [inline tests/test_decoder.py]: the constants are strings?

Comment (jcc333) [inline tests/test_decoder.py]: nm, I take your meaning

Comment (mrgaaron) [inline tests/test_decoder.py]: The point of defining the constant is to hide that detail from the user.  If we are going to expose the detail anyway then we can delete the constants

**Pull request merged**